packages/config/config/devices/0x0234/zif5020_2.0.json

```diff
@@ -0,0 +1,229 @@
+			"label": "Threshold value for activation of inputs.",
+			"unit": "200 ms",
+			"defaultValue": 20
+		},
+		"13": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "Threshold for input in latched mode.",
+			"unit": "50 ms",
+			"defaultValue": 50
+		},
+		"14": {
+			"$import": "templates/logic_group_template.json#enable_central_scene",
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Enable",
+					"value": 0
+				},
+				{
+					"label": "Disable",
+					"value": 1
+				}
+			]
+		},
+		"15[0x01]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 2 secure message",
+			"valueSize": 2
+		},
+		"15[0x02]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 3 secure message",
+			"valueSize": 2
+		},
+		"15[0x04]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 4 secure message",
+			"valueSize": 2
+		},
+		"15[0x08]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 5 secure message",
+			"valueSize": 2
+		},
+		"15[0x10]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 6 secure message",
+			"valueSize": 2
+		},
+		"15[0x20]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 7 secure message",
+			"valueSize": 2
+		},
+		"15[0x40]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 8 secure message",
+			"valueSize": 2
+		},
+		"15[0x80]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 9 secure message",
+			"valueSize": 2
+		},
+		"15[0x100]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 10 secure message",
+			"valueSize": 2
+		},
+		"15[0x0200]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 11 secure message",
+			"valueSize": 2
+		},
+		"15[0x0400]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 12 secure message",
+			"valueSize": 2
+		},
+		"15[0x0800]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association group 13 secure message",
+			"valueSize": 2
+		},
+		"15[0x0fff]": {
+			"label": "Messages in all association groups are sent as secure",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4095,
+			"defaultValue": 4095,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 4095
+				}
+			]
+		}
```

Comment:
No overlaps please

Reply:
This can be removed from the code. references overlapping bit masks